dotnet/aspnetcore/containerWithTests/Application/aspnet-core-dotnet-core.FunctionalTests/SampleFunctionalTests.cs

```diff
@@ -36,8 +36,9 @@ public void TestClean()
         public void SampleFunctionalTest1()
```

Comment:
we are not using this sample anymore. We are likely using aspnetcore22 now

Reply:
We have this sample deployed currently, shall I close the PR, if we are using aspnetcore22 from now?

Reply:
let me check

Reply:
let me check